client/components/Content/Content.jsx

```diff
@@ -0,0 +1,45 @@
+import { h } from "preact";
+import cx from "classnames";
+
+import PrettyPrint from "../PrettyPrint";
+
+import styles from "./content.css";
+
+const Content = ({ data }) => {
+  if (!data || !data.length) {
+    return;
+  }
+  return (
+    <div className={styles.content}>
+      <div className={styles.printWrapper}>
+        {(() => {
```

Comment:
Ah I had used this for a different purpose, but now its not required, I will remove it.